starknet_devnet/blueprints/rpc/schema.py

```diff
@@ -147,7 +150,9 @@ def assert_valid_rpc_request(*args, method_name: str, **kwargs):
 
         for name, schema in schemas.items():
             if name not in kwargs:
-                raise ValidationError(f"Missing kwarg for {name}.")
+                if schema["is_required"]:
+                    raise ValidationError(f"Missing kwarg for {name}.")
+                continue
```

Comment:
Do we need `continue` here?

Reply:
If there's no value for kwarg and it is not required, we should just skip the value and continue validating next kwarg.